lib/api-util.js

```diff
@@ -16,8 +16,11 @@ const HTTPError = sUtil.HTTPError;
  */
 function mwApiGet(app, domain, query) {
 
-    query = query || {};
-    query.continue = query.continue || '';
+    query = Object.assign({
+        format: 'json',
+        formatversion: 2,
+        continue: ''
```

Comment:
I'm not sure about the empty continue by default. Why are we doing that?

Reply:
Honestly... I think this is just old habit.  :)  There was a period around 2014-15 in which the MW API was issuing warnings if neither a 'continue' or 'rawcontinue' parameter was supplied with api.php?action=query.  In clients (inc. the Android app) we just started adding '&continue=' to every query.  I'll take it out. 

https://lists.wikimedia.org/pipermail/mediawiki-api-announce/2014-September/000069.html